.github/workflows/ios-e2e-test.yml

```diff
@@ -2,7 +2,7 @@ name: Test iOS e2e
 on:
   pull_request:
     paths:
-        - 'Example/**'
+      - 'Example/**'
```

Comment:
A question that should've been to previous PRs but I couldn't do a review in time before merging - does this mean e2e tests are only run on PRs when the `Example/` project changes? Shouldn't it also be run when `ios/` and `android/` are being changed? cc @WoLewicki

Reply:
Yeah, for sure these tests should be run for such changes.

Reply:
It takes about 40 minutes for tests to run. I am not sure if we want them ran on every change in most PRs. They will always be run on the master after merging. Also one can ran them locally if they are not sure if something works

Reply:
Only master option seems enough for me.

Reply:
Okay, running e2e on every commit is overkill but if it's possible we could maybe run these tests 1 time a day (in the night) on a hanging PR? If it's not possible it's okay (not too sure if GitHub Actions supports these kind of tricks). Our master is not directly production so it should be fine as you've said.

Reply:
Unfortunately we can always do cron actions on master branch (or other specified in the workflow).